turfpy/misc.py

```diff
@@ -48,19 +49,6 @@ def line_intersect(
     >>> l2 = Feature(geometry=LineString([[123, -18], [131, -14]]))
     >>> line_intersect(l1, l2)
     """
-    if not compat.HAS_GEOPANDAS:
-        raise ImportError(
-            "line_intersect requires `Spatial indexes` for which it "
-            "requires `geopandas` and either `rtree` or `pygeos`. "
-            "See installation instructions at https://geopandas.org/install.html"
-        )
-    elif not compat.HAS_PYGEOS and not compat.HAS_RTREE:
-        raise ImportError(
-            "line_intersect requires `Spatial indexes` for which it "
-            "requires `geopandas` and either `rtree` or `pygeos`. "
-            "See installation instructions at https://geopandas.org/install.html"
-        )
-    import geopandas  # noqa
```

Comment:
any reason to remove these checks?

Reply:
Geopandas is a requirement now by default. Pygeos is no longer maintained.